docs/Technical-Documentation/Architecture-Decision-Record/020-pipeline-build-flow

```diff
@@ -0,0 +1,44 @@
+# 20. Pipeline Build Flow
```

Comment:
can you make this file a `.md`?

Reply:
Oops, yeah that was the intent. TY for the catch

Reply:
done